CHANGELOG.md

```diff
@@ -1,109 +1,99 @@
-## 0.10.x
+# 0.10.x
```

Comment:
Will do! Honestly, I might have accidentally hit backspace on that line. I don't remember doing that!